### ISSUE TITLE
adding the vita3k emulator to emudeck

### ISSUE THREAD

**Issue by The-Math-God** [dragoonDorise/EmuDeck#336]
is there any chance of adding the vita3k emulator to emudeck 
I mean it already has an arch Linux version 
and there are some great games that are playable 
like Muramasa rebirth for example

**Comment by Godsbane** [comment]
Hi - i investigated this emulator a bit, and it's on the list of things i'd like to add. Right now there's not a good way to parse with steam rom manager (because files are installed inside the vita data structures instead of being loaded externally like other systems)

EmulationStation-DE has a way to handle this via text files with the game id's inside. This would be fairly manual as far as processes go. You would have to install the game in vita3k, and then makea  file with the name of your game in the vita folder, and then put the game id inside that file.

TLDR; yes, we should add this emu. Just don't quite know the right way yet.

**Comment by Godsbane** [comment]
added to beta
